src/map.js

```diff
@@ -163,14 +164,19 @@ export default class GpxMap {
         line.addTo(this.map);
 
         this.tracks.push(line);
+    }
 
+    recenter() {
         if (!this.scrolled)
             this.center();
     }
 
     center() {
+        if (this.tracks.length == 0)
+            return;
```

Comment:
If no GPX files can be successfully loaded, then the `center` function will be triggered: it should do nothing.

Reply:
Could you also add this as a comment?

Reply:
443e466e3513891ef4bff9c5029d4e63c64d09ea